lux/compiler/Compiler.py

```diff
@@ -111,6 +114,8 @@ def expandUnderspecified(ldf, viewCollection):
 		ldf : lux.luxDataFrame.LuxDataFrame
 			LuxDataFrame with underspecified context
 
+		viewCollection : list[lux.view.View]
+			empty list that will be populated with specified lux.View objects.
```

Comment:
This is not always empty right?

Reply:
Yeah, I think I blindly copy and pasted. Updated.